File: externals/Async-CTP/samples/VB/Windows/Phone/Concert/Finder/Concert/Finder/ViewModel/ArtistDetailsPageViewModel.vb
Namespace ConcertFinder.ViewModel

    ''' <summary>
    ''' The view model for the ArtistDetailsPage.
    ''' </summary>
    Public Class ArtistDetailsPageViewModel
        Inherits AbstractDetailsPageViewModel

        Public Sub New()
            Me.EventGroup = Model.EventGroup.Artist
        End Sub

    End Class

End Namespace

File: externals/Async-CTP/samples/CS/Windows/Phone/Concert/Finder/Concert/Finder/ViewModel/ArtistDetailsPageViewModel.cs
using ConcertFinder.Model;

namespace ConcertFinder.ViewModel
{
    /// <summary>
    /// The view model for the ArtistDetailsPage.
    /// </summary>
    public class ArtistDetailsPageViewModel : AbstractDetailsPageViewModel
    {
        public ArtistDetailsPageViewModel()
        {
            this.EventGroup = EventGroup.Artist;
        }
    }
}
